OpenDreamRuntime/Procs/Native/DreamProcNativeHelpers.cs

```diff
@@ -463,4 +470,18 @@ internal static partial class DreamProcNativeHelpers {
 
     [GeneratedRegex("[\\^]|[^a-z0-9@]")]
     private static partial Regex CkeyRegex();
+
+    /// <summary>
+    /// Returns one of NORTH, SOUTH, EAST, WEST, NORTHEAST, NORTHWEST, SOUTHEAST, or SOUTHWEST
+    /// </summary>
+    public static AtomDirection GetRandomDirection(DreamManager dreamManager) {
+        AtomDirection[] dirs = [
```

Comment:
should this not be a static const outside the proc def? idk if C# is smart enough to not redefine the list every time the proc is called